package.json

```
@@ -84,8 +83,8 @@
     "focus-trap": "^5.1.0"
   },
   "peerDependencies": {
+    "prop-types": "^15.7.2",
     "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "prop-types": "^15.7.2"
+    "react-dom": "^16.0.0"
```

I don't think we should force our consumers to now also upgrade `react-dom`. Perhaps we can just widen this?

```suggestion
    "react-dom": ">=15.0.0"
```

---

The only reason the diff is pulling this out is because of the removal of the coma, since the `prop-types` peer was moved above in alphabetical order. This PR isn't suggesting that we update the version of `react-dom`.

Also, we've actually already published a major version of `focus-trap-react` (well, I did 🙂 , I think that was just before you came on, back on the original repo before the move to this org) with `react-dom: ^16.0.0`. React 15 is super old now. I didn't see any point in making a statement that any changes going forward would actually work all the way back to React 15. I think it's already quite a statement to say we'd support back to 16.0.